### DISCUSSION TITLE
This will be a great addition to Alfred workflow.

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [ramandv/obsidian-snippets-manager#2] [Ideas]
Hi,
If we can create an Alfred workflow to quickly search and copy snippets without opening obsidian, this can be useful for programming to quickly search for snippets, and paste in the opened editor/IDE.

**Comment by ramandv** [comment]
@PKDmeDEV New version has been released with Alfred integration. I was thinking about this too.
Check it out and let me know.

**Reply by piyushdongre-dot-com** [reply]
Hi, 
I just reinstalled it, and now I can see an option for a toggle button. Could you please elaborate on how to make it work?

**Reply by piyushdongre-dot-com** [reply]
Never mind, saw the updated documentation and it works. Kudos..
I used to use codespace.app for such a thing. Now all my snippets can live happily in Obsidian along my notes.

**Reply by piyushdongre-dot-com** [reply]
Hi,

Just noticed one thing. The switch needs to be toggled again and again after adding new snippets. Can we make this automatic?

**Reply by ramandv** [reply]
Whenever you try to search snippets in obsidian, at that time, it will reload all the snippets file.
It will update the data for Alfred json also at that time.

**Reply by piyushdongre-dot-com** [reply]
Ok. As my usecase is generally out of Obsidian. So if searching generates the alfred-snippets.json, I can attach a shortcut key to the snipper search, and that will do it.

Thanks

**Reply by ramandv** [reply]
@PKDmeDEV Snippet manager is especially useful in mobile to get the data that we wanted. You can create URI shortcut to the search directly in mobile. Otherwise Alfred was ok for me too ;-)

**Reply by piyushdongre-dot-com** [reply]
Hi again, 

Little offtopic from this plugin, but can we modify the Alfred workflow 
https://github.com/chrisgrieser/shimmering-obsidian?tab=readme-ov-file#installation

to copy notes to the clipboard instead of opening it. Then, we may not need to scope the snippets to a folder.

**Comment by ramandv** [comment]
Sorry I don't understand. I don't use this plugin though
